widget/accordion.go

```diff
@@ -155,9 +161,10 @@ func (r *accordionRenderer) Layout(size fyne.Size) {
 }
 
 func (r *accordionRenderer) MinSize() (size fyne.Size) {
+	divHeight := 1
```

Comment:
There should only be one authoritative answer to the question for `divHeight`. It should be extracted into a constant.

Reply:
Indeed, corrected